src/__tests__/type.js

```diff
@@ -1066,3 +1066,59 @@ test('navigation key: {arrowleft} and {arrowright} moves the cursor', () => {
     input[value="abc"] - keyup: c (99)
   `)
 })
+
+test('{arrowdown} fires keyup/keydown events', () => {
+  const {element} = setup('<input />')
+  const handleKeyDown = jest.fn()
+  const handleKeyUp = jest.fn()
+
+  addListeners(element, {
+    eventHandlers: {
+      keyDown: handleKeyDown,
+      keyUp: handleKeyUp,
+    },
+  })
```

Comment:
Any reason we're using the `addListeners` util here instead of the `getEventSnapshot` as above? Gives us a better idea of all events fired.

I think this would be better for both tests.

Reply:
Will do! I've made this change for both tests.